VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,245 @@
+
+
+Let’s take a concrete example to illustrate the aforementioned concept. In this example, there are 10 validators with equal stake. Each of them applies a 1% commission and has 20% of self-bonded ATOMs. We consider a successful block collects a total of 1025.51020408 ATOMs in fees.
+
+First, a 2% tax is applied.
+* 2% * 1025.51020408 = 20.51020408 ATOMs go to the reserve pool
+
+1005 ATOMs now remain. Let’s assume that the proposer included 100% of the signatures in its block. It thus obtains the full bonus of 5%.
+
+We have to solve this simple equation to find the reward R for each validator:
+
+`9*R + R + R*5% = 1005 ⇔ R = 1005/10.05 = 100`
+
+* For the proposer validator
+  * The pool obtains R + R*5% = 105 ATOMs
+  * Commission = 105*80%*1% = 0.84 ATOMs
+  * Validator reward = 100 * 20% + Commission = 21.84 ATOMs
+  * Delegators rewards = 105 * 80% - Commission = 83.16 ATOMs (each delegator will be able to claim its portion of these rewards in proportion of their stake)
+* For each non-proposer validator
+  * The pool obtains R = 100 ATOMs
+  * Commission = 100*80%*1% = 0.8 ATOMs
+  * Validator reward = 100 * 20% + Commission = 20.8 ATOMs
+  * Delegators rewards = 100 * 80% - Commission = 79.2 ATOMs (each delegator will be able to claim its portion of these rewards in proportion of their stake)
+
+### What are the slashing conditions?
+
+If a validator misbehaves, its bonded stake along with its delegator’s stake and will be slashed. The severity of the punishment depends on the type of fault. There are 3 main faults that can result in loss of funds for a validator and its delegators :
+
+* **Double signing:** If someone reports on chain A that a validator signed two blocks at the same height on chain A and chain B, this validator will get slashed on chain A
+* **Unavailability:** If a validator’s signature has not been included in the last X block, the validator will get slashed by a marginal amount proportional to X. If X is above a certain limit Y, then the validator will get unbonded
+* **Non-voting:** If a validator did not vote on a proposal and once the fault is reported by a someone, his stake will be a little slashed. He will also get a penalty. Once a validator gets a penalty, no additional penalty can be submitted for a given period (3 weeks). After this period, if a second penalty is submitted, the validator gets a second light slash and gets unbonded. If, after the first penalty, no other penalty is submitted for a 3 months period, then the penalty can be cleared by sending a special transaction.
```

Comment:
their stake will receive a minor slash

Reply:
REMOVE all use of his/he in this paragraph

Reply:
addressed